index.html

```diff
@@ -33,8 +33,8 @@ <h2 class="guideTitle">GUIDELINES:</h2>
     <div class="main">
         <!-- Start Template Div -->
         <div class="box">
-            <div id="yourName"></div>
-            <p id="nameP">By <a href="https://github.com/SushilBajracharya01/">yours</a></p>
+            <div id="Prakhar"></div>
```

Comment:
Hello Prakhar, Thank you for sending a PR to this repo, I really appreciate it, but there is a bit of a problem in the code, you are supposed to make a copy of the Template Div and work on it, not change the Template div itself. So could you let the template div be unchanged and work on the copy of it instead? and also in the CSS file, #yourName styles should not be changed. Thank you.

Reply:
oh okay, I understood it completely. Sorry for the inconvenience, I'll change it accordingly.
Thanks

Reply:
Its alright, It happens sometimes. I like your art, Nice work.